ports/mimxrt/boards/MIMXRT1020_EVK/mpconfigboard.mk

```diff
@@ -16,6 +13,19 @@ else
 JLINK_CONNECTION_SETTINGS =
 endif
 
+CFLAGS += \
+	-DUSDHC_AVAIL=1
+
+SRC_C += \
+	sdcard.c \
+	machine_sdcard.c \
+	hal/flexspi_nor_flash.c
+
+SRC_QSTR += \
+	machine_sdcard.c
+
+SRC_HAL_IMX_C += \
+	$(MCU_DIR)/drivers/fsl_usdhc.c
```

Comment:
can this also be added unconditionally to the Makefile?  the idea is to make board configuration as simple as possible

Reply:
fixed.

Reply:
I changed the configuration to a makefile variable. Using that we can control adding the define and the addition of  `fsl_usdhc.c`. Unfortunately I can not unconditionally add this file since it is not present for the MIMXRT1010, for instance.